docs/make.jl

```diff
@@ -0,0 +1,27 @@
+using ADNLPModels
+using Documenter
+using Literate
```

Comment:
Why do we need Literate?

Reply:
I was first planning to use, and finally decided against it and forgot to remove it.
Thanks for the feedback !